init/gravitino/gravitino-dependency.sh

```diff
@@ -17,26 +17,22 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-# set -ex
-playground_dir="$(dirname "${BASH_SOURCE-$0}")"
-playground_dir="$(cd "${playground_dir}">/dev/null; pwd)"
-isExist=`which docker-compose`
-if [ $isExist ]
-then
-  true # Placeholder, do nothing
-else
-  echo "ERROR: No docker service environment found, please install docker-compose first."
-  exit
-fi
+gravitino_dir="$(dirname "${BASH_SOURCE-$0}")"
+gravitino_dir="$(
+  cd "${gravitino_dir}" >/dev/null
+  pwd
+)"
+. "${gravitino_dir}/../common/common.sh"
 
-components=""
-case "${1}" in
-  *)
-    components=$@
-esac
+# Prepare download packages
+if [[ ! -d "${gravitino_dir}/packages" ]]; then
+  mkdir -p "${gravitino_dir}/packages"
+fi
```

Comment:
I didn't find any package that needs to be downloaded in here.

Reply:
I move the "# Prepare download packages" comment to the packages URL